### PR TITLE
Update s4sdkjs-odata-service-cloud-foundry from QA to Production

### PR DESCRIPTION
**Pull request by mr-flannery**
This PR introduces the API Business Hub Sandbox as explicit alternative to the Mockserver for  testing the OData VDM.

I also added a bunch of switchable plain JS code snippets.

### CODE REVIEW

File: tutorials/s4sdkjs-odata-service-cloud-foundry/s4sdkjs-odata-service-cloud-foundry.md [hunk 1]
```diff
@@ -14,14 +14,28 @@ primary_tag: products>sap-s-4hana-cloud-sdk
...
 
 Once it is up and running you should see the list of services at `http://localhost:3000/`.
 
+Alternatively, many APIs can also be tested using the sandbox of the SAP API Business Hub. To use the sandbox, you need an an API key. Go to [https://api.sap.com](https://api.sap.com) and click "Log On" in the top right corner. If you do not have an account, you will need to register first. When you are logged in, click on "Hi <your name>" in the top right corner and then click on "Preferences" in the dropdown menu that just opened. On the preferences page, click on "Show API Key".
+
+To use the sandbox for your requests, you will need to pass the API key to the VDM requests using the `withCustomHeaders` method, and you will need to add the correct URL to your destinations. Checkout the following example:
```
Comment (sacnl) [inline tutorials/s4sdkjs-odata-service-cloud-foundry/s4sdkjs-odata-service-cloud-foundry.md]: Shouldn't this part go to the place where the API is being called for the first time? That is, in step 4 when explaining how to use basic authentication.

Comment (mr-flannery) [reply tutorials/s4sdkjs-odata-service-cloud-foundry/s4sdkjs-odata-service-cloud-foundry.md]: I see what you mean. I will update it and then ping you again.

Comment (mr-flannery) [reply tutorials/s4sdkjs-odata-service-cloud-foundry/s4sdkjs-odata-service-cloud-foundry.md]: Please re-check.

**Comment by mr-flannery** [comment]
Heads-up: since PRs always go from the fork's master to this master, the scope of this PR changed.

File: tutorials/cloudsdk-js-vdm-update/cloudsdk-js-vdm-update.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Update OData Entities with the SAP Cloud SDK's Virtual Data Model
-description: Update OData entities with the SAP Cloud SDK's virtual data model to build an address manager application.
+description: Update OData dntities with the SAP Cloud SDK's virtual data model to build an address manager application.
```
Comment (sacnl) [inline tutorials/cloudsdk-js-vdm-update/cloudsdk-js-vdm-update.md]: Typo: entities

**Pull request merged**